<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalcWindow</class>
 <widget class="QMainWindow" name="CalcWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalcWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="7" column="1">
     <widget class="QPushButton" name="but0">
      <property name="font">
       <font>
        <family>PT Serif</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
	
	background-color: rgb(0, 0, 0);
	padding : 15px;
	color: rgb(255, 255, 127);
}

QPushButton::pressed 
{
	background-color : rgb(255,255,127);
	padding : 15px;
	color : rgb(0,0,0);
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="butAdd">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>PT Serif</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{	
	background-color: rgb(244, 244, 244);
	padding : 15px;	
	color: rgb(85, 85, 255);
}

QPushButton::pressed 
{
	background-color : #f4f4f4;
	padding : 15px;
	color: rgb(85, 85, 255);
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="but1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>PT Serif</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
	
	background-color: rgb(0, 0, 0);
	padding : 15px;
	color: rgb(255, 255, 127);
}

QPushButton::pressed 
{
	background-color : rgb(255,255,127);
	padding : 15px;
	color : rgb(0,0,0);
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="butSub">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>PT Serif</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{	
	background-color: rgb(244, 244, 244);
	padding : 15px;	
	color: rgb(85, 85, 255);
}

QPushButton::pressed 
{
	background-color : #f4f4f4;
	padding : 15px;
	color: rgb(85, 85, 255);
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="but2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>PT Serif</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
	
	background-color: rgb(0, 0, 0);
	padding : 15px;
	color: rgb(255, 255, 127);
}

QPushButton::pressed 
{
	background-color : rgb(255,255,127);
	padding : 15px;
	color : rgb(0,0,0);
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3" rowspan="4">
     <widget class="Line" name="line">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 255);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="butDot">
      <property name="font">
       <font>
        <family>PT Serif</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{	
	background-color: rgb(244, 244, 244);
	padding : 15px;	
	color: rgb(85, 85, 255);
}

QPushButton::pressed 
{
	background-color : #f4f4f4;
	padding : 15px;
	color: rgb(85, 85, 255);
}</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="butDiv">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>PT Serif</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{	
	background-color: rgb(244, 244, 244);
	padding : 15px;	
	color: rgb(85, 85, 255);
}

QPushButton::pressed 
{
	background-color : #f4f4f4;
	padding : 15px;
	color: rgb(85, 85, 255);
}</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="but6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>PT Serif</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
	
	background-color: rgb(0, 0, 0);
	padding : 15px;
	color: rgb(255, 255, 127);
}

QPushButton::pressed 
{
	background-color : rgb(255,255,127);
	padding : 15px;
	color : rgb(0,0,0);
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="6">
     <widget class="QLineEdit" name="Display">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Myriad Pro</family>
        <pointsize>48</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 127);
background-color: rgb(0,0,0);
margin-top : -3px;
border: none;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QPushButton" name="butPow">
      <property name="font">
       <font>
        <family>PT Serif</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{	
	background-color: rgb(244, 244, 244);
	padding : 15px;	
	color: rgb(85, 85, 255);
}

QPushButton::pressed 
{
	background-color : #f4f4f4;
	padding : 15px;
	color: rgb(85, 85, 255);
}</string>
      </property>
      <property name="text">
       <string>^</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="but3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>PT Serif</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
	
	background-color: rgb(0, 0, 0);
	padding : 15px;
	color: rgb(255, 255, 127);
}

QPushButton::pressed 
{
	background-color : rgb(255,255,127);
	padding : 15px;
	color : rgb(0,0,0);
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QPushButton" name="butMul">
      <property name="font">
       <font>
        <family>PT Serif</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{	
	background-color: rgb(244, 244, 244);
	padding : 15px;	
	color: rgb(85, 85, 255);
}

QPushButton::pressed 
{
	background-color : #f4f4f4;
	padding : 15px;
	color: rgb(85, 85, 255);
}</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="but7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>PT Serif</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
	
	background-color: rgb(0, 0, 0);
	padding : 15px;
	color: rgb(255, 255, 127);
}

QPushButton::pressed 
{
	background-color : rgb(255,255,127);
	padding : 15px;
	color : rgb(0,0,0);
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="but5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>PT Serif</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
	
	background-color: rgb(0, 0, 0);
	padding : 15px;
	color: rgb(255, 255, 127);
}

QPushButton::pressed 
{
	background-color : rgb(255,255,127);
	padding : 15px;
	color : rgb(0,0,0);
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QPushButton" name="butBack">
      <property name="font">
       <font>
        <family>PT Serif</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{	
	background-color: rgb(244, 244, 244);
	padding : 15px;	
	color: rgb(85, 85, 255);
}

QPushButton::pressed 
{
	background-color : #f4f4f4;
	padding : 15px;
	color: rgb(85, 85, 255);
}</string>
      </property>
      <property name="text">
       <string>&lt;-</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="but4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>PT Serif</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
	
	background-color: rgb(0, 0, 0);
	padding : 15px;
	color: rgb(255, 255, 127);
}

QPushButton::pressed 
{
	background-color : rgb(255,255,127);
	padding : 15px;
	color : rgb(0,0,0);
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="but9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>PT Serif</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
	
	background-color: rgb(0, 0, 0);
	padding : 15px;
	color: rgb(255, 255, 127);
}

QPushButton::pressed 
{
	background-color : rgb(255,255,127);
	padding : 15px;
	color : rgb(0,0,0);
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="butClr">
      <property name="font">
       <font>
        <family>PT Serif</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{	
	background-color: rgb(244, 244, 244);
	padding : 15px;	
	color: rgb(85, 85, 255);
}

QPushButton::pressed 
{
	background-color : #f4f4f4;
	padding : 15px;
	color: rgb(85, 85, 255);
}</string>
      </property>
      <property name="text">
       <string>CLR</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="but8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>PT Serif</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{
	
	background-color: rgb(0, 0, 0);
	padding : 15px;
	color: rgb(255, 255, 127);
}

QPushButton::pressed 
{
	background-color : rgb(255,255,127);
	padding : 15px;
	color : rgb(0,0,0);
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="6" column="5" rowspan="2">
     <widget class="QPushButton" name="butAns">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>PT Serif</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton 
{	
	background-color: rgb(244, 244, 244);
	padding : 15px;	
	color: rgb(85, 85, 255);
}

QPushButton::pressed 
{
	background-color : #f4f4f4;
	padding : 15px;
	color: rgb(85, 85, 255);
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="6">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 170, 127);
background-color: rgb(0, 0, 0);
margin-left: -10px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
